app/assets/javascripts/avaya-webchat-2.js

```diff
@@ -47,14 +47,7 @@
             avayaWebchatClose.hidden = true;
             avayaWebchatMaintenance.hidden = true;
             if(ccState === 'Open'){
-                if(availableAgents > 0 && ewt <= 300){
-                    avayaWebchatOpen.hidden = false;
-                }else{
-                    avayaAgentBusy.hidden = false;
-                }
-            }else{
-               avayaWebchatClose.hidden = false;
-            }
+                avayaWebchatOpen.hidden = false;
```

Comment:
We still need 
 }else{
               avayaWebchatClose.hidden = false;
            }

Reply:
added back in